api/wasm-interpreter/Cargo.toml

```diff
@@ -28,6 +28,9 @@ js-sys = "0.3.44"
 wasm-bindgen-futures = { version = "0.4.18" }
 wasm-bindgen = { version = "0.2.66" }
 
+# Enable image-rs' default features to make all image formats available for the wasm interpreter
+image = { version = "0.24.0" }
```

Comment:
Wait...  Don't we load the image with the <img> element on wasm.  I don't think we need that then.

Reply:
Ah that's true. Good point! I'll remove that again.